src/svg/opentable.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
You should pass this file into https://jakearchibald.github.io/svgomg/ first. It will do some cleanup.